ymir/command/mir/commands/copy.py

```diff
@@ -80,29 +80,11 @@ def run_with_args(mir_root: str,
 
         PhaseLoggerCenter.update_phase(phase='copy.read')
 
-        need_change_class_ids = True
-        unknown_names_and_count = {}
-        if drop_annotations:
-            mir_annotations.prediction.Clear()
-            mir_annotations.ground_truth.Clear()
-            need_change_class_ids = False
-        if data_mir_root == mir_root:
-            need_change_class_ids = False
-
-        if need_change_class_ids:
-            src_class_id_mgr = class_ids.load_or_create_userlabels(label_storage_file=data_label_storage_file)
-            dst_class_id_mgr = class_ids.load_or_create_userlabels(label_storage_file=label_storage_file)
-            src_to_dst_ids = {
-                src_class_id_mgr.id_and_main_name_for_name(n)[0]: dst_class_id_mgr.id_and_main_name_for_name(n)[0]
-                for n in src_class_id_mgr.all_main_names()
-            }
-
-            CmdCopy._change_type_ids(single_task_annotations=mir_annotations.prediction, src_to_dst_ids=src_to_dst_ids)
-            CmdCopy._change_type_ids(single_task_annotations=mir_annotations.ground_truth,
-                                     src_to_dst_ids=src_to_dst_ids)
-            unknown_names_and_count = CmdCopy._gen_unknown_names_and_count(src_class_id_mgr=src_class_id_mgr,
-                                                                           mir_context=mir_context,
-                                                                           src_to_dst_ids=src_to_dst_ids)
+        unknown_names_and_count = copy_annotations(mir_annotations=mir_annotations,
+                                                   mir_context=mir_context,
+                                                   drop_annotations=drop_annotations,
```

Comment:
drop_annotations can be processed forehead.

Reply:
done